src/main/java/ui/windows/StatsDisplayData.java

```diff
@@ -78,22 +58,22 @@ public int getGamesWon() {
      * @param gamesWon the number of games won as an integer
      */
     public void setGamesWon(int gamesWon) {
-        this.gamesWon = gamesWon;
+        this.gamesLost = gamesWon;
```

Comment:
This should not have been deleted; the new code sets the gamesLost value as the gamesWon, but it should just stay as it was.

Reply:
Actually, this method should be **set**GamesWon(), not get.

Reply:
The method name is setGamesWon().